### PR TITLE
Replace National Historic Trail shields with triangular color swatches

### PR DESCRIPTION
**Pull request by 1ec5**
For compliance reasons, the shields introduced in #1115, #1116, and #1117 have been replaced with simple color swatches. Each color swatch is in the shape of a Reuleaux triangle, a shape that is common to many recreational trails, including U.S. Bicycle Routes. The colors continue to correspond to the primary color associated with each trail.

### CODE REVIEW

**Comment by quincylvania** [comment]
I'm afraid using plain shields for NHTs would be too confusing. A few of the NHTs are overlapping, so users would need to refer to a legend and be able to distinguish between similar colors to tell which is which. I suggest either making our own designs or removing these entirely until we figure out if we can use the official emblems.

**Comment by 1ec5** [comment]
These shields do appear in the legend, and I’ve seen maps that take the same approach to simplifying the shields. But if the concern is about potential color conflicts or colorblindness issues, I suppose we could stick a monogram inside each shield.

File: icons/shield_us_nht_semo.svg
Comment (1ec5) [inline icons/shield_us_nht_semo.svg:1]: Also, this fixes a path issue in the original image that had a bit of blue showing through on the other side of the white ring.

**Review by ZeLonewolf — APPROVED** [review]
Temporary pause until we figure out what's permitted.

File: icons/shield_us_nht_semo.svg
Comment (quincylvania) [reply icons/shield_us_nht_semo.svg:1]: The dark blue intentionally overlaps the white part slightly, as per the original shield.

**Comment by quincylvania** [comment]
Yeah, I guess a solid color is better than nothing for now :)

File: icons/shield_us_nht_semo.svg
Comment (1ec5) [reply icons/shield_us_nht_semo.svg:1]: I was referring to this itty-bitty portion, which admittedly is too small to show up at 1×:

A light blue, triangular gap between the white and black rings.

**Pull request merged**